app/client/components/payment/update/ContactUs.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { css } from "@emotion/core";
+import { textSans } from "@guardian/src-foundations/typography";
+import { neutral, brand } from "@guardian/src-foundations/palette";
+import { minWidth } from "../../../styles/breakpoints";
+import { space } from "@guardian/src-foundations";
+
+const privacyNoticeLinkCss = css`
+  color: ${brand[500]};
+  text-decoration: underline;
+`;
+
+const ContactUs = () => (
+  <p
+    css={css`
+      width: 100%;
+      border-top: 1px solid ${neutral[86]};
+      ${textSans.medium()};
+      color: ${neutral[46]};
+      padding-top: ${space[4]}px;
+
+      ${minWidth.tablet} {
+        padding-top: ${space[9]}px;
+      }
+    `}
+  >
+    Are you experiencing difficulties switching your payment method?{" "}
+    <a css={privacyNoticeLinkCss} href="/help-centre">
+      Contact Us &nbsp; &nbsp;
```

Comment:
Can this be:

Contact Us{"  "}

to make it more consistent with line 27 for example?

Reply:
its not working with multiple whitespaces

Reply:
Rather than inlining an SVG could you not use the `src-link` component and the `SvgChevronRightSingle` icon?
https://guardian.github.io/source/?path=/docs/source-src-link-link--playground

(I know the docs are for `v3`, but I've used in my live chat branch so the API seems to be the same for `v2.8.2`)